clang/test/CXX/drs/cwg1884.cpp

```diff
@@ -0,0 +1,548 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: split-file --leading-lines %s %t
+// RUN: %clang_cc1 -std=c++20 -pedantic-errors -fexceptions -fcxx-exceptions %t/cwg1884_A.cppm -triple x86_64-unknown-unknown -emit-module-interface -o %t/cwg1884_A.pcm
+// RUN: %clang_cc1 -std=c++20 -verify=since-cxx20 -pedantic-errors -fexceptions -fcxx-exceptions -triple x86_64-unknown-unknown %t/cwg1884.cpp -fmodule-file=cwg1884_A=%t/cwg1884_A.pcm
```

Comment:
```suggestion
// RUN: %clang_cc1 -std=c++20 -verify -pedantic-errors -fexceptions -fcxx-exceptions -triple x86_64-unknown-unknown %t/cwg1884.cpp -fmodule-file=cwg1884_A=%t/cwg1884_A.pcm
```

This is a DR to c++98 (that we can't test in older language modes)

Reply:
I don't want readers of the test to have a misconception that this is tested in C++17 and earlier modes. Even you just paid attention to the fact that this DR, which is applied to C++98, is only tested in C++20 mode and newer. So I consider using `since-cxx20` prefix a feature.